attributes/web.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Cookbook Name:: stackstorm
+# Attributes:: web
+#
+
+# Self-signed certificate
+default['stackstorm']['ssl']['self_signed']['enabled'] = false
```

Comment:
It would be nice to go with the following naming structure for `st2web` package: 
`default['stackstorm']['web']...`

In future this will make more sense, when we'll add `st2chatops` package:
`default['stackstorm']['chatops']...`